doc/source/workflow.md

```diff
@@ -0,0 +1,29 @@
+# Typical Workflows
+
+## List Available Datasets
+
+Datafiles are organized into ensembles, which collect all data needed for a specific purpose. To view a list of all datasets in a particular ensemble, query the `multimeta` API. The `multimeta` API gives each datafile as a unique identifier string and a collection of attributes describing the data contained within that file. After a datafile of interest has been determined from its metadata attributes, its unique identifier string may be used to request the data.
```

Comment:
Hyperlink to `multimeta` documentation?

Reply:
I would love to! I haven't figured out how yet. [It's not just me](https://github.com/miyakogi/m2r/issues/41).